contents/javascript/Javascript-modules.md

```diff
@@ -71,13 +71,13 @@ ES6 modules supports advanced features such as _[asynchronous loading](http://ex
 
 Due to the very recent adoption of ES6 modules by browsers and there is still some browsers that do not support it, you might not be able to use ES6 modules right away in those unsupported browsers.
```

Comment:
Could you also modify the line here to 'manageable'?

https://github.com/se-edu/learningresources/blame/93c24e7d4da32a78f80091cf05bfc3fc195792a3/contents/javascript/Javascript-modules.md#L30

Reply:
rely on a -> relies on a 
https://github.com/se-edu/learningresources/blame/93c24e7d4da32a78f80091cf05bfc3fc195792a3/contents/javascript/Javascript-modules.md#L39

Reply:
there is -> there are

https://github.com/se-edu/learningresources/blame/93c24e7d4da32a78f80091cf05bfc3fc195792a3/contents/javascript/Javascript-modules.md#L49

https://github.com/se-edu/learningresources/blame/93c24e7d4da32a78f80091cf05bfc3fc195792a3/contents/javascript/Javascript-modules.md#L72

Reply:
Thanks! I initially only meant to change the outdated or incorrect js stuff, I'll go ahead and change these other minor issues too

Reply:
Done!